{
	"header": {
		"line": "CVE-2021-3429 (When instructing cloud-init to set a random password for a new user ac ...)",
		"name": "CVE-2021-3429",
		"description": "(When instructing cloud-init to set a random password for a new user ac ...)"
	},
	"annotations": [
		{
			"line": "\t{DLA-2601-1}",
			"type": "xref",
			"bugs": [
				"DLA-2601-1"
			]
		},
		{
			"line": "\t- cloud-init 20.4.1-2 (bug #985540)",
			"type": "package",
			"package": "cloud-init",
			"kind": "fixed",
			"version": "20.4.1-2",
			"flags": [
				{
					"bug": 985540
				}
			]
		},
		{
			"line": "\t[buster] - cloud-init 20.2-2~deb10u2",
			"type": "package",
			"release": "buster",
			"package": "cloud-init",
			"kind": "fixed",
			"version": "20.2-2~deb10u2"
		},
		{
			"line": "\tNOTE: https://github.com/canonical/cloud-init/commit/b794d426b9ab43ea9d6371477466070d86e10668",
			"type": "NOTE",
			"description": "https://github.com/canonical/cloud-init/commit/b794d426b9ab43ea9d6371477466070d86e10668"
		}
	]
}
